cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1086,8 +1086,10 @@ def _add_head_node(self):
                     "cfn_fsx_options": self._get_shared_storage_options(SharedStorageType.FSX),
                     "cfn_volume": self._get_shared_storage_ids(SharedStorageType.EBS),
                     "cfn_scheduler": self._cluster_config.scheduling.scheduler,
-                    "cfn_encrypted_ephemeral": head_node.storage.ephemeral_volume.encrypted
-                    if head_node.storage and head_node.storage.ephemeral_volume
+                    "cfn_encrypted_ephemeral": "true"
+                    if head_node.storage
+                    and head_node.storage.ephemeral_volume
+                    and head_node.storage.ephemeral_volume.encrypted
```

Comment:
Minor: it seems we need to proceed to include all implied sections to avoid such a long statement here?

Reply:
Yes, we'll discuss about this. good point.